kube/secret.go

```diff
@@ -0,0 +1,101 @@
+package kube
+
+import (
+	"fmt"
+	//	"strconv"
+	"strings"
+
+	"github.com/SUSE/fissile/model"
+	meta "k8s.io/client-go/pkg/api/unversioned"
+	apiv1 "k8s.io/client-go/pkg/api/v1"
+)
+
+// NewSecret creates a single new, empty K8s Secret
+func NewSecret(name string) *apiv1.Secret {
+	secret := &apiv1.Secret{
+		TypeMeta: meta.TypeMeta{
+			APIVersion: "v1",
+			Kind:       "Secret",
+		},
+		ObjectMeta: apiv1.ObjectMeta{
+			Name: name,
+		},
+	}
+	secret.Data = make(map[string][]byte)
+	return secret
+}
+
+// Ref is an entry in the RefMap
+type Ref struct {
```

Comment:
`SecretRef` (and below, `RefMap` → `SecretRefMap`)? Otherwise it's not obvious (from other files) this has anything to do with secrets.